xml/networking-octavia_admin.xml

```diff
@@ -403,6 +403,164 @@ Updated port: 0b0301b9-4ee8-4fb6-a47c-2690594173f4</screen>
+   <procedure>
+     <title>Manually deleting load balancers created with octavia</title>
+     <step>
+       <para>
+         Query the Octavia service for the loadbalancer ID:
+       </para>
+       <screen>
+&prompt.ardana;openstack loadbalancer list --column id --column name --column provisioning_status
++--------------------------------------+---------+---------------------+
+| id                                   | name    | provisioning_status |
++--------------------------------------+---------+---------------------+
+| d8ac085d-e077-4af2-b47a-bdec0c162928 | test-lb | ERROR               |
++--------------------------------------+---------+---------------------+
+       </screen>
+     </step>
+     <step>
+       <para>
+         Query the Octavia service for the amphora IDs (in this
+         example we use ACTIVE/STANDBY topology with 1 spare Amphora):
+       </para>
+       <screen>
+&prompt.ardana;openstack loadbalancer amphora list
++--------------------------------------+--------------------------------------+-----------+--------+---------------+-------------+
+| id                                   | loadbalancer_id                      | status    | role   | lb_network_ip | ha_ip       |
++--------------------------------------+--------------------------------------+-----------+--------+---------------+-------------+
+| 6dc66d41-e4b6-4c33-945d-563f8b26e675 | d8ac085d-e077-4af2-b47a-bdec0c162928 | ALLOCATED | BACKUP | 172.30.1.7    | 192.168.1.8 |
+| 1b195602-3b14-4352-b355-5c4a70e200cf | d8ac085d-e077-4af2-b47a-bdec0c162928 | ALLOCATED | MASTER | 172.30.1.6    | 192.168.1.8 |
+| b2ee14df-8ac6-4bb0-a8d3-3f378dbc2509 | None                                 | READY     | None   | 172.30.1.20   | None        |
++--------------------------------------+--------------------------------------+-----------+--------+---------------+-------------+
+       </screen>
+     </step>
+     <step>
+       <para>
+         Query the Octavia service for the loadbalancer pools:
+       </para>
+       <screen>
+&prompt.ardana;openstack loadbalancer pool list
++--------------------------------------+-----------+----------------------------------+---------------------+----------+--------------+----------------+
+| id                                   | name      | project_id                       | provisioning_status | protocol | lb_algorithm | admin_state_up |
++--------------------------------------+-----------+----------------------------------+---------------------+----------+--------------+----------------+
+| 39c4c791-6e66-4dd5-9b80-14ea11152bb5 | test-pool | 86fba765e67f430b83437f2f25225b65 | ACTIVE              | TCP      | ROUND_ROBIN  | True           |
++--------------------------------------+-----------+----------------------------------+---------------------+----------+--------------+----------------+
+       </screen>
+     </step>
+     <step>
+       <para>
+         Connect to the <literal>octavia</literal> database:
```

Comment:
I don't think octavia needs to be in a `<literal>` tag here. Is there a reason why?

Reply:
No particular reason really. I'll remove it.